x/ibc-account/module.go

```diff
@@ -191,7 +188,7 @@ func (am AppModule) OnChanCloseInit(
 	channelID string,
 ) error {
 	// Disallow user-initiated channel closing for interchain account channels
-	return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "user cannot close channel")
+	return nil
```

Comment:
This will allow anyone to close a channel. If you want to let people voluntarily close a channel, you need to do at least some kind of authentication here

Reply:
Updated this :+1: